src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -149,12 +149,16 @@ private function registerAnnotationLoaders(ContainerBuilder $container)
     private function registerFileLoaders(ContainerBuilder $container)
     {
         $paths = [];
+        $globOptions = GLOB_BRACE | GLOB_NOSORT;
+        $prefix = DIRECTORY_SEPARATOR.'Resources'.DIRECTORY_SEPARATOR.'config'.DIRECTORY_SEPARATOR;
```

Comment:
I can but I wanted to avoid adding a dependency only for this.
Le mar. 7 juin 2016 à 12:07, Kévin Dunglas <notifications@github.com> a
écrit :

> In src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php
> <https://github.com/api-platform/core/pull/568#discussion_r66043028>:
>
> > @@ -149,12 +149,16 @@ private function registerAnnotationLoaders(ContainerBuilder $container)
> >      private function registerFileLoaders(ContainerBuilder $container)
> >      {
> >          $paths = [];
> > +        $globOptions = GLOB_BRACE | GLOB_NOSORT;
> > +        $prefix = DIRECTORY_SEPARATOR.'Resources'.DIRECTORY_SEPARATOR.'config'.DIRECTORY_SEPARATOR;
>
> What do you think about using the Symfony Finder Component instead of glob
> directly?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/api-platform/core/pull/568/files/fd99357da41448ed77aceac668050c0b1b7d3ad8#r66043028>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABQr81YXxO09RMmxY-MAwF1LJ_-k7Z6Dks5qJULFgaJpZM4Ivt1f>
> .
>